src/modules/openrcdmcryptcfg/main.py

```diff
@@ -20,6 +20,12 @@
                         languages=libcalamares.utils.gettext_languages(),
                         fallback=True).gettext
 
+unencrypted_separate_boot = False
```

Comment:
This looks really out-of-place: it's indented wrong, and is outside of any function. I would expect it to live inside the `write_dmcrypt_conf()` function.

Reply:
Should be fixed now